services/testing/nightwatch/main.go

```diff
@@ -23,56 +69,112 @@ func calcSiteURL() string {
+	fmt.Println(Yellow(Sprintf("    If you want to run this command directly, you need run it from %s", Bold("inside the Tokaido SSH container"))))
+
+	ssh.StreamConnectCommand([]string{"cd", cp, "&&", "yarn", "test:nightwatch"})
 
-	console.Println("👩‍💻  Tokaido is starting a Nightwatch test run with the command `yarn test:nightwatch`\n", "")
-	utils.StreamOSCmdContext(d, "yarn", "test:nightwatch")
 }
 
 func checkCompatibility() error {
 	goos.CheckDependencies()
 	return nil
 }
 
-// RunDrupalTests - Run Drupal Nightwatch tests. See: https://github.com/drupal/drupal/blob/8.6.x/core/tests/README.md#nightwatch-tests
-func RunDrupalTests() error {
-	err := CheckLocalDrupal()
+func enableChromedriver() {
+	if !conf.GetConfig().Services.Chromedriver.Enabled {
+		conf.SetConfigValueByArgs([]string{"services", "chromedriver", "enabled", "true"}, "project")
+	}
+
+	if !docker.StatusCheck("chromedriver", conf.GetConfig().Tokaido.Project.Name) {
+		fmt.Println(Cyan("👾  Restarting Tokaido with Chromedriver enabled"))
+		tok.Init(true, false)
+
+	}
+}
+
+// checkMysqlTestsDatabase spawns a 'tests' databases in the Tokaido MySQL instance
+func checkMysqlTestsDatabase() bool {
+	// Check to see if the database already exists
+	db, err := database.Connect("tests")
 	if err != nil {
-		console.Println("⚠️   "+err.Error(), "")
-		return err
+		fmt.Println(Red("Error: Could not connect to the database container failed. Have you run `tok up`?"))
+		utils.DebugString(err.Error())
+		os.Exit(1)
 	}
 
-	err = checkCompatibility()
+	err = db.Ping()
 	if err != nil {
-		return err
+		// The database connection was opened but failed, so the provided database doesn't exist
+		return false
 	}
 
-	checkPHPUnit()
-	yarnInstall()
+	return true
+}
 
-	configureEnvFile()
+func createMysqlTestsDatabase() {
+	fmt.Println(Cyan("    Creating a new 'tests' database"))
 
-	yarnTestNightwatch()
+	db, err := database.ConnectRoot("tokaido")
+	if err != nil {
+		fmt.Println(Red("Error: Could not connect to the database container failed. Have you run `tok up`?"))
+		utils.DebugString(err.Error())
+		os.Exit(1)
+	}
 
-	return nil
+	// Create the database
+	utils.DebugString("Creating test database")
+	_, err = db.Exec("CREATE DATABASE tests;")
+	if err != nil {
+		fmt.Println(Red("Error: Could not create the tests database: "), err.Error())
+		utils.DebugString(err.Error())
+		os.Exit(1)
+	}
+
+	// Grant 'tokaido' user access
+	utils.DebugString("Creating full access to 'tests' database to 'tokaido' user")
+	_, err = db.Exec("GRANT ALL ON tests.* TO 'tokaido'@'%';")
+	if err != nil {
+		fmt.Println(Red("Error: Could not grant access to the tests database: "), err.Error())
+		utils.DebugString(err.Error())
+		os.Exit(1)
+	}
+
+	// Flush
+	utils.DebugString("Flushing privileges")
+	_, err = db.Exec("FLUSH PRIVILEGES;")
+	if err != nil {
+		fmt.Println(Red("Error: Could not successfully flush privileges: "), err.Error())
+		utils.DebugString(err.Error())
+		os.Exit(1)
+	}
```

Comment:
I can imagine all this working around things would have been a pain to get to this point!

Reply:
I'm sure there's more to come!